app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,36 @@
+ 
+class AdminsController < ApplicationController
+    #prepend_before_action :require_no_authentication, only: [:cancel]
+    before_action :authorize_admin
+    # skip_before_action :verify_signed_out_user 
+    
+    def index
+        @users = User.all
+    end
+    
+    def edit
+        @user = User.find(params[:id])
+    end
+    
+    def update 
+        #This gets the info from the edit form and updates the data. 
+        @user = User.find(params[:id])
+        @user.email = params[:user][:email]
```

Comment:
Here strong parameters should be used, so that you don't need a line for each individual attribute.  there was previous discussion on this -- Marcus, let me know if you have questions

Reply:
I've been able to update using strong params, and got around the password issues. I'll have this in the next commit/pull/review request.